samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj

```diff
@@ -51,8 +51,8 @@
     <!-- Custom Fonts -->
     <MauiFont Include="Resources\Fonts\*" />
 
-    <PackageReference Include="Microsoft.Maui.Controls" Version="[$(MauiPackageVersion),)"/>
-    <PackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="[$(MauiPackageVersion),)"/>
+    <PackageReference Include="Microsoft.Maui.Controls" Version="8.0.21" />
```

Comment:
Is the root cause a MAUI bug that is fixed in 8.0.21? If so, we'll want to bump the `MauiPackageVersion` variable in `Directory.Build.props`.

Reply:
But we want to be wary of when `8.0.21` will ship in Visual Studio based on the last community standup

Reply:
@brminnick, yes, thanks, you're right. I just sent a commit with changes to Directory.Build.props

Reply:
Thanks @olekssokol!

@bijington - yup! I'll add the `blocked` tag so that we don't accidentally merge this PR until 8.0.24 is included in the Visual Studio (stable) release.